Docker/Dockerfile

```diff
@@ -0,0 +1,183 @@
+ADD installKNITR.r /app
+
+# Enviornment variables
+ENV clinsv_path "/app/clinsv"
+
+# Move all contents of clinsv scripts to the clinsv folder
+RUN mv clinsvScripts/* clinsv/
+RUN rm -r clinsvScripts
+
+ENV PATH="/app/clinsv/bin:${PATH}"
+
+# Get links to repos
+RUN apt-get update
+
+# To stop essential asking for input
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Australia/Sydney
+
+RUN apt-get -y install build-essential
+
+RUN apt-get -y install wget curl git nano
+
+# Dependencies for Tabix and Big perl modules
+RUN apt-get -y install zlib1g-dev 
+
+# Samtools dependencies
+RUN apt-get -y install libbz2-dev liblzma-dev libcurl4-openssl-dev libncurses5-dev
+
+# Python 3.8 dependencies
+RUN apt-get -y install python3.8 python3-pip
+
+# Root v6 dependencies
+RUN apt-get -y install dpkg-dev cmake g++ gcc binutils libx11-dev libxpm-dev libxft-dev libxext-dev libssl-dev
+
+# Lumpy dependencies
+RUN apt-get -y install dh-autoreconf
+
+# Change sh command to use bash rather than dash from bin.
+RUN ln -sf ../bin/bash ../bin/sh
+
+# Install cpanm
+RUN curl -L https://cpanmin.us | perl - App::cpanminus
+
+RUN mkdir clinsv/perl
+
+# Install Bio::DB::Big perl module
+RUN cd $clinsv_path/perl && \
+    git clone https://github.com/dpryan79/libBigWig && \
+    cd libBigWig/ && \
+    make && \
+    export LIBBIGWIG_DIR=$PWD && \
+    cpanm install Bio::DB::Big
+
+# Install samtools bcftools and htslib 
+RUN cd $clinsv_path && \
+    mkdir samtools && \
+    cd samtools && \
+    wget https://github.com/samtools/samtools/releases/download/1.4/samtools-1.4.tar.bz2 && \
+    wget https://github.com/samtools/bcftools/releases/download/1.4/bcftools-1.4.tar.bz2 && \
+    wget https://github.com/samtools/htslib/releases/download/1.4/htslib-1.4.tar.bz2 && \
+    tar xvfj samtools-1.4.tar.bz2 && \
+    tar xvfj bcftools-1.4.tar.bz2 && \
+    tar xvfj htslib-1.4.tar.bz2 && \
+    cd htslib-1.4 && \
+    make && \ 
+    make prefix=$clinsv_path/samtools install && \
+    cd ../samtools-1.4 && \
+    make && \ 
+    make prefix=$clinsv_path/samtools install && \
+    cd ../bcftools-1.4 && \
+    make && \
+    make prefix=$clinsv_path/samtools install && \
+    cd .. && \
+    rm -r htslib* samtools* bcftools*
+
+# Create symbolic links to all the samtools  tools
+RUN cd $clinsv_path/bin && \
+    ln -s ../samtools/bin/bcftools bcftools && \
+    ln -s ../samtools/bin/samtools samtools && \
+    ln -s ../samtools/bin/bgzip bgzip && \
+    ln -s ../samtools/bin/tabix tabix 
+
+# Install bigwigtools
+RUN mkdir $clinsv_path/bigWigTools && \
+    cd $clinsv_path/bigWigTools && \
+    wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/bigWigToWig && \
+    wget http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64.v369/wigToBigWig && \
+    chmod ug+x $clinsv_path/bigWigTools/* 
+
+RUN cd $clinsv_path/bin && \
+    ln -s ../bigWigTools/bigWigToWig bigWigToWig && \
+    ln -s ../bigWigTools/wigToBigWig wigToBigWig
+
+# Install python2
```

Comment:
do we still need py2?

Reply:
I did have issues trying to run py3 with everything. see [here](https://github.com/KCCG/ClinSV/issues/13#issuecomment-973557541) . Could give it another go...